ctrd/supervisord/config.go

```diff
@@ -23,8 +23,6 @@ type Config struct {
 	OOMScore int `toml:"oom_score"`
 	// Cgroup specifies cgroup information for the containerd daemon process
 	Cgroup CgroupConfig `toml:"cgroup"`
-
-	md toml.MetaData
```

Comment:
cc @fuweid

Reply:
don't remove this part @ZYecho Thanks

Reply:
The code is copied from containerd as I comment. I want to keep it the same with containerd.